modules/example/src/smithy4s/example/FancyList.scala

```diff
@@ -1,11 +1,12 @@
 package smithy4s.example
 
 import smithy4s.Newtype
+import smithy4s.example.refined.FancyList._
```

Comment:
Example of the import I am referring to in my previous comment

Reply:
See [2ebcd04](https://github.com/disneystreaming/smithy4s/pull/346/commits/2ebcd046176466ced908069e4287221546f371a0)